apps/mobile/src/locales/en/messages.po

```diff
@@ -172,19 +172,19 @@ msgstr "Avatar"
 #: src/components/create-new-wallet/mnemonic-display.tsx:13
 #: src/components/recover-wallet/recover-wallet-sheet.tsx:28
 msgid "BIP39 passphrase"
-msgstr "BIP39 passphrase<<<<<<< HEAD"
+msgstr "BIP39 passphrase"
 
 #: src/components/widgets/tokens/tokens.mocks.tsx:50
 msgid "Bitcoin"
 msgstr "Bitcoin"
 
 #: src/components/widgets/tokens/tokens.mocks.tsx:27
 msgid "Bitcoin blockchain"
-msgstr "Bitcoin blockchain>>>>>>> bbb518c (feat: add tokens widget, ref leather-io/issues#221)"
+msgstr "Bitcoin blockchain"
```

Comment:
I keep getting these weird rebase issues in the locales file. It's annoying and takes up time fixing.

I wonder how to avoid?

Reply:
It is annoying, I just delete the files and run `pnpm lingui` in mobile root to regenerate the files.

Reply:
I haven't been deleting them, I'll do that in future and look into adding a shortcut to do that. Thanks!